libraries/resource_hab_sup.rb

```diff
@@ -42,13 +44,31 @@ class HabSup < Resource
 
         hab_package 'core/hab-sup' do
           bldr_url new_resource.bldr_url if new_resource.bldr_url
+          version hab_version
+        end
+
+        hab_package 'core/hab-launcher' do
+          bldr_url new_resource.bldr_url if new_resource.bldr_url
+          version launcher_version
         end
       end
 
       action_class do
+        HAB_VERSION = '0.59.0'.freeze
```

Comment:
This version pin replicates the pin defined in https://github.com/chef-cookbooks/habitat/blob/master/resources/install.rb#L86. If we're going to be pinning to that version in multiple places, I'd like us to extract that somewhere to be shared between all locations which need to pin.